genderbias/femalewords/__init__.py

```diff
@@ -1,32 +1,41 @@
-from genderbias.detector import Detector, Flag, Issue, Report
 import os
 import re
 
+from ..detector import Detector, Flag, Issue, Report
+
 _dir = os.path.dirname(__file__)
-FEMALE_WORDS = [word.strip() for word in open(_dir + "/Femalewords.wordlist", 'r').readlines()]
+FEMALE_WORDS = [
+    word.strip() for word in open(_dir + "/Femalewords.wordlist", "r").readlines()
+]
+
 
 class FemaleDetector(Detector):
-    
+    """
+    A detector for words that tend to be used more frequently for recruiting
+    women than for recruiting men.
+
+    TODO: Move this into `genderedwords` detector
```

Comment:
This line illustrates why I don't think we should have `TODO`s in the code. There is already a `genderedwords` detector, and it does something completely different from this detector. So the comment is confusing, and it might lead to someone breaking another detector that currently works. 

A short comment doesn't explain enough, and a long comment would be a distraction from the code that exists now. Future work should be explained/explored in issues, not the code.